infra/config-server/src/main/resources/config-repo/order-service-dev.yml

```
@@ -42,4 +42,8 @@ service:
   jwt:
     access-expiration: 3600000
     # secret-key: ${SECRET_KEY}
-    secret-key: "401b09eab3c013d4ca54922bb802bec8fd5318192b0a75f201d8b3727429080fb337591abd3e44453b954555b7a0812e1081c39b740293f765eae731f5a65ed1"
\ No newline at end of file
+    secret-key: "401b09eab3c013d4ca54922bb802bec8fd5318192b0a75f201d8b3727429080fb337591abd3e44453b954555b7a0812e1081c39b740293f765eae731f5a65ed1"
+
+admin:
+  # token: ${ADMIN_TOKEN}
+  token: "AcwNIiPfouAQU0TgGRwDC0a7UUtbu8nvUhIh3M+xf+I="
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)